packages/lib-react-components/src/helpers/withResponsiveContext/README.md

```diff
@@ -0,0 +1,18 @@
+# withResponsiveContext
+
+A HOC which accepts a component and returns the component
+wrapped in Grommet's `ResponsiveContext` component, with the size passed into
+wrapped component's `size` prop.
+
+## Usage
+
+```js
+
+class MyComponent extends React.Component {
+  render () {
+    ...
```

Comment:
A quick example might be handy, especially to illustrate the fact that `ResponsiveContext` returns breakpoint names rather than pixel values